### ISSUE TITLE
Sound effects for common code components

### ISSUE THREAD

**Issue by samreid** [phetsims/wave-interference#458]
@Ashton-Morris provided audio for the stopwatch and measuring tape, we are waiting for https://github.com/phetsims/tambo/issues/73 to be further along before integrating the sounds.

**Comment by samreid** [comment]
@Ashton-Morris or @emily-phet does anything specific remain for this issue that isn't already done or covered elsewhere?

**Comment by samreid** [comment]
@emily-phet says we may want more common code sound effects, but not for the next version, so deferring.

**Comment by samreid** [comment]
After the commit, the sim now has basic sounds for radio buttons, checkboxes, buttons, drag and drop things, etc. I'll request review of this for correctness/completeness in a later phase.

**Comment by arouinfar** [comment]
Thanks @samreid. Overall Basic Sounds were working well, but you missed the BarriersNode. Claude Code helped me hook it up and it worked in local testing, and was commited in https://github.com/phetsims/totality/commit/b21b4588ee610f5d91353a5397a54565478b8cf1. The commit links this issue, so not sure why it isn't automatically showing up.

Here's a summary of the changes made to `wave-interference/js/slits/view/BarriersNode.ts`:

**Problem:** The barrier on the Slits screen had no sound at all — it used a plain `DragListener` and `AccessibleSlider` with no sound configuration, unlike other interactive elements in the sim.

**Changes:**

1. **Replaced `DragListener` with `SoundDragListener`** — gives the barrier grab and release sounds when dragged with a pointer, consistent with other draggable items in the sim.

2. **Added a `ValueChangeSoundPlayer`** wired into the `AccessibleSlider` `drag` callback — plays a pitch-mapped sound on each keyboard step when using arrow/Page keys, giving auditory feedback proportional to the barrier's position within its range.

The two input modalities are intentionally different: pointer gets grab/release only (value-change sounds on top of that felt redundant), while keyboard gets value-change sounds since there's no grab/release equivalent.

Over to @samreid to review the commit and close if all is well.

**Comment by samreid** [comment]
Yes, that is the correct implementation and it sounds good at runtime, thanks! I'm adding other sounds in https://github.com/phetsims/wave-interference/issues/541 related to alt-input, and this one is ready to close.
